src/vs/workbench/api/browser/mainThreadWebviewPanels.ts

```diff
@@ -170,7 +171,8 @@ export class MainThreadWebviewPanels extends Disposable implements extHostProtoc
 			title: initData.title,
 			options: reviveWebviewOptions(initData.panelOptions),
 			contentOptions: reviveWebviewContentOptions(initData.webviewOptions),
-			extension
+			extension,
+			codeWindow: getActiveWindow()
```

Comment:
This can be potentially a request from extensions (say from a command), `getActiveWindow()` is probably the best guess.

Reply:
Reading `EditorGroupColumn` is right. I'm not sure though how to read the `targetGroup` resolved from the showOptions above, its type is `IEditorGroup | GroupIdentifier | SIDE_GROUP_TYPE | ACTIVE_GROUP_TYPE | AUX_WINDOW_GROUP_TYPE`, for `IEditorGroup` or `GroupIdentifier`, it should be sufficient. For `SIDE_GROUP_TYPE` or `ACTIVE_GROUP_TYPE`, is it active window or just main window? Lastly, `AUX_WINDOW_GROUP_TYPE` is for new aux window, where the window id is unknown? I wonder if there is already a service/helper that help me find the right window id from this info. @bpasero

Reply:
Unfortunately resolving the right group needs the input already being there because it depends on that.